gusto/physics/boundary_and_turbulence.py

```diff
@@ -141,7 +142,9 @@ def __init__(self, equation, T_surface_expr, vapour_name=None,
                 dmv_expr = surface_expr * (mv_np1_expr - m_v) / self.dt
                 source_mv_expr = test_m_v * self.source_mv * dx
 
-                self.source_interpolators.append(Interpolator(dmv_expr, self.source_mv_int))
+                self.source_interpolators.append(
+                    lambda expr=dmv_expr: assemble(interpolate(dmv_expr, self.source_theta_vd_int), tensor=self.source_mv_int)
```

Comment:
```suggestion
                    lambda: assemble(interpolate(dmv_expr, self.source_theta_vd_int), tensor=self.source_mv_int)
```
?

Reply:
This is the same lambda query. I can believe that I was wrong in this suggestion.